packages/app-mobile/components/Dropdown.js

```diff
@@ -38,10 +38,9 @@ class Dropdown extends React.Component {
 		const listTop = Math.min(maxListTop, this.state.headerSize.y + this.state.headerSize.height);
 
 		const wrapperStyle = {
-			width: this.state.headerSize.width,
 			height: listHeight + 2, // +2 for the border (otherwise it makes the scrollbar appear)
 			marginTop: listTop,
-			marginLeft: this.state.headerSize.x,
+			alignSelf: 'center',
```

Comment:
I don't have much confidence that this change is not going to break something. What was the reason for changing this?

Reply:
`marginLeft` was used to shift the dropdown if some buttons were active but now since buttons are already hidden so no need to shift.
`width` is also used to change width based on current width which was earlier depending on buttons but now it will take full width.
This is a simple styling change, so I don't think it will break.